spring-boot/proxy/src/main/java/eu/coatrack/proxy/security/AdminCommunicator.java

```diff
@@ -0,0 +1,88 @@
+package eu.coatrack.proxy.security;
+
+/*-
+ * #%L
+ * coatrack-proxy
+ * %%
+ * Copyright (C) 2013 - 2021 Corizon | Institut für angewandte Systemtechnik Bremen GmbH (ATB)
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import eu.coatrack.api.ApiKey;
+import eu.coatrack.api.ServiceApi;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+/**
+ * Offers communication services to the Coatrack admin server to receive data
+ * required by the gateway for key verification.
+ *
+ * @author Christoph Baier
+ */
+
+@Service("adminCommunicator")
+public class AdminCommunicator {
+
+    private static final Logger log = LoggerFactory.getLogger(eu.coatrack.proxy.security.AdminCommunicator.class);
```

Comment:
No, it is not required. This is probably be a leftover because I worked with the "Resource" annotation before using constructor injections. This will be removed.